coordinator/src/services/translation-service.ts

```diff
@@ -468,7 +468,7 @@ export function convertFhirMessageToHl7V3ParentPrescription(fhirMessage: fhir.Bu
     const root = {
         ParentPrescription: convertBundleToParentPrescription(fhirMessage)
     }
-    const options = {compact: true, ignoreComment: true, spaces: 4}
+    const options = {compact: true, ignoreComment: true, spaces: 4, attributesFn: sortAttributes} as unknown as XmlJs.Options.JS2XML
```

Comment:
This is a bit odd, it would be nice to understand whats going on here. Is there a reason we need this change as part of this PR?

Reply:
The casting is due to the imported type definitions not matching the underlying types providing false positive errors at compilation time. This solution has been applied elsewhere in the code. The sortAttributes function has been added to ensure that xml attributes are parsed in alphabetical order so that the generated integration test response can be reliably checked